NEWS.md

```diff
@@ -1,3 +1,8 @@
+### v8.17.1 (2022-08-02)
+ * grpc: bound the external client segment to the onReceiveStatus listener to propagate transaction context to the grpc client callbacks.
```

Comment:
This could be worded slightly more from the customer standpoint so they know what problem it is solving but I think it can go as-is, particularly since we are fixing this so fast.

Reply:
yea i was unclear the right way to represent this.  I think we should fix but not sure how, thoughts?

Reply:
Maybe something like this?

* Fixed issue where instrumetned code invoked within a `@grpc/grpc-js` client callback would not get tracked by the agent.

  Bound the external client segment to the onReceiveStatus listener to propagate transaction context to the grpc client callbacks.